### ISSUE TITLE
Google Tag Manager iOS

### ISSUE THREAD

**Issue by MaximeLassalleHub** [arnesson/cordova-plugin-firebase#1100]
Hello there,

I'm setting up Google Tag Manager with the plugin in a Ionic 3 app.
The plugin version I use is the last release 2.0.5.

> cordova-plugin-firebase 2.0.5 "Google Firebase Plugin"

For both iOS & Android, Firebase is running correctly.
On Android, I've been able to setup Google Tag Manager thanks to the log the Sdk gave me.
My config.xml for Android :
    

> <platform name="android">
>             <resource-file src="GTM-XXXX.json" target="app/src/main/assets/containers/GTM-XXXX.json" />

...

For iOS, I have nothing helping me to know if it's working or not, as no log is given.
My iOS config:
    

> <platform name="ios">
>         <resource-file src="GTM-XXXX.json" />
> ...

I pretty sure it's not doing anything with the container file on iOS, but still not 100% sure.

Anyone can help me to finish the iOS Google Tag Manager setup, and/or verify the configuration is successful? 

Thank you,

Regards,

**Comment by phiasco12** [comment]
did you find any solution to this?

**Comment by MaximeLassalleHub** [comment]
Nope. I ended implementing GTM into the Javascript layer, not the native ones.

**Comment by bitflower** [comment]
@MaximeLassalleHub did you have to config anything special to make it work on iOS? Especially regarding the protocol?

**Comment by billysean** [comment]
any news on this one? i have the need to implement GTM firebase so that i can send across data to google analytics

**Comment by teslavitas** [comment]
@MaximeLassalleHub Did you encounter an issue with sending collect requests to the Google Analytics?
